jsonnet/thanos-ruler.jsonnet

```diff
@@ -161,11 +160,11 @@ local authorizationRole =
 
       service:
         service.new(
-          'thanos-ruler-' + thanosRulerConfig.name,
+          'thanos-ruler',
           thanosRulerConfig.selectorLabels,
           [
-            ports.newNamed('grpc', thanosRulerConfig.ports.grpc, 'grpc'),
```

Comment:
The grpc endpoint is still necessary for the connection towards Thanos Querier, no?

Reply:
IIUC it will be needed for Thanos Querier -> Ruler requests but we're not there yet. Also it was overridden by `service.mixin.spec.withPorts(...)` at L183 so it was a no-op in practice (proof is that it didn't show up in `assets/thanos-ruler/service.yaml`).